gcs/package.json

```diff
@@ -6,7 +6,7 @@
     "name": "Avis-Drone-Labs"
   },
   "private": true,
-  "version": "0.1.6-alpha",
+  "version": "alpha-0.1.7",
```

Comment:
Why did you switch this?

Reply:
oh whoops I did mean to switch that back (was messing around with version names), it does look a little nicer tho. Idm switching it back

Reply:
Think switch it back, it's been that way all the previous releases so we shouldn't change it